src/script/components/TitleBar/TitleBar.tsx

```diff
@@ -144,6 +144,12 @@ export const TitleBar: React.FC<TitleBarProps> = ({
   const {setCurrentView: setView} = useAppMainState(state => state.responsiveView);
 
   const setLeftSidebar = () => setView(ViewType.LEFT_SIDEBAR);
```

Comment:
`setLeftSidebar`is also used when clicking the call button and shouldn't close the right sidebar in that case so an ` onClickBack`function is added